interfaces/builtin/basedeclaration.go

```diff
@@ -217,6 +217,11 @@ slots:
       slot-snap-type:
         - core
     deny-auto-connection: true
+  download:
+    allow-installation:
+      slot-snap-type:
+        - app
+    deny-auto-connection: false
```

Comment:
@jdstrand should this be `true` so that it can be overridden with a snap declaration assertion from the store?